modeldb/modeldb-run.yaml

```diff
@@ -1219,6 +1229,7 @@
     comment: Tries to open bac6.ses, which does not exist
     run: null
 267384:
+    github: 'pull/2'
```

Comment:
This one should be merged imminently; it slipped through the cracks when the others went in.